README.md

```diff
@@ -120,6 +120,7 @@ subjective "from more to less liquid control-ly" order.
 | :--                | :-- | --: | :-: |
 | AIO liquid cooler  | [Corsair Hydro H80i GT, H100i GTX, H110i GTX](docs/asetek-690lc-guide.md) | <sup>_Ze_</sup> | 1.9.1 |
 | AIO liquid cooler  | [Corsair Hydro H80i v2, H100i v2, H115i](docs/asetek-690lc-guide.md) | <sup>_Z_</sup> | 1.9.1 |
+| AIO liquid cooler  | [Corsair Hydro H110i GT](docs/coolit-guide.md) | <sup>_e_</sup> | git |
```

Comment:
We've mostly done away with the `(experimental)` tagging for devices in #663, so `e` is not needed. Same goes for the docs and name of the device in the driver.

Reply:
Done